### ISSUE TITLE
multiple values for capitalization in .editorconfig no longer works since VS 17.14.0

### ISSUE THREAD

**Issue by astaerk** [dotnet/roslyn#78574]
**Version Used**: VS 17.14.0

We have a large code base where some private methods are camelCase and some private methods are PascalCase. 
We tried to decide for one or the other but some designers will generate camelCase methods and some PascalCase methods and so in the end we gave up renaming all of them and decided to allow both cases for private methods.

Among other things our .editorconfig contains the folowing rules:
```
# Naming rules
dotnet_naming_rule.private_methods_should_be_pascal_or_camel_case.severity = warning
dotnet_naming_rule.private_methods_should_be_pascal_or_camel_case.symbols = private_methods
dotnet_naming_rule.private_methods_should_be_pascal_or_camel_case.style = pascal_or_camel_case

# Symbol specifications
dotnet_naming_symbols.private_methods.applicable_kinds = method
dotnet_naming_symbols.private_methods.applicable_accessibilities = private
dotnet_naming_symbols.private_methods.required_modifiers = 

# Naming styles
dotnet_naming_style.pascal_or_camel_case.required_prefix = 
dotnet_naming_style.pascal_or_camel_case.required_suffix = 
dotnet_naming_style.pascal_or_camel_case.word_separator = 
dotnet_naming_style.pascal_or_camel_case.capitalization = camel_case, pascal_case
```

Up until Visual Studio 17.13 this was working fine. We didn't get warnings for our private methods no matter if they are camel or pascal case.
But after we updated to Visual Studio 17.14.0 we get style violations for our private methods.

**Steps to Reproduce**:
1. checkout https://github.com/proj-arch/EditorconfigNamingRuleErrorDemo
2. open in Visual Studio 17.14.0 and have a look at the warnings
**OR**
1. create a new project
2. add an .editorconfig file with the rules stated above
3. open in Visual Studio 17.14.0 and have a look at the warnings

**Diagnostic Id**:

IDE1006 - Naming rule violation: These words must begin with upper case characters: privateCamelCaseMethod

**Expected Behavior**:
No warnings are shown.

**Actual Behavior**:
The following warning is shown: IDE1006 - Naming rule violation: These words must begin with upper case characters: privateCamelCaseMethod
